android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -210,9 +202,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setUpToolbar();
         setUpNavDrawer();
-        setUpProgressBar();
         setUpCustomTab();
 
+        completeHandler = DownloadCompleteHandler.with(context);
```

Comment:
This is fluent API

When not attaching detaching much, could be used as

````Java
DownloadCompleteHandler.with(context)
    .startListening()
    .show()
    .withCompleteListener()
    .subscribe();
````

The full effect of its ease is not visible as there are lots of `show` and `hide` called on it, needing to separate the object